app/jobs/check_no_induction_participants_job.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+class CheckNoInductionParticipantsJob < ApplicationJob
+  def perform
+    ECFParticipantEligibility.where(no_induction: true).find_each do |eligibility|
+      revalidate(eligibility.participant_profile)
+    end
+  end
+
+private
+
+  def revalidate(participant_profile)
+    validation_data = participant_profile.ecf_participant_validation_data
+
+    validation_attributes =
+      validation_data.attributes.symbolize_keys.slice(
+        :trn, :nino, :full_name, :date_of_birth
+      )
+
+    dqt_response = ParticipantValidationService.validate(
+      config: {
+        check_first_name_only: true,
+      },
+      **validation_attributes,
+    )
+
+    validation_attributes[:dob] = validation_attributes.delete(:date_of_birth)
+
+    StoreValidationResult.call(
+      participant_profile: participant_profile,
+      validation_data: validation_attributes,
+      dqt_response: dqt_response,
+    )
```

Comment:
🤦 For some reason I didn't think that would work... I didn't see that you could not pass in data and it would default to the saved data. I was planning on extracting this into a revalidation service to dry it up when we need to revalidate other users